headers/functions/arm9.h

```diff
@@ -90,13 +90,13 @@ void FileSeek(struct file_stream* file, int offset, int whence);
 void FileClose(struct file_stream* file);
 void UnloadFile(void* ptr);
 void LoadFileFromRom(struct iovec* iov, const char* filepath, uint32_t flags);
-uint32_t GetDebugFlag1(uint32_t flag_id);
-void SetDebugFlag1(uint32_t flag_id, uint32_t val);
+bool GetDebugFlag(enum debug_flag flag);
+void SetDebugFlag(enum debug_flag flag, bool val);
 int AppendProgPos(char* str, struct prog_pos_info* prog_pos, const char* msg);
 void DebugPrintTrace(const char* msg, struct prog_pos_info* prog_pos);
 void DebugPrint0(const char* fmt, ...);
-uint32_t GetDebugFlag2(uint32_t flag_id);
-void SetDebugFlag2(uint32_t flag_id, uint32_t val);
+bool GetDebugLogFlag(enum debug_flag flag);
```

Comment:
Should this be:

```suggestion
bool GetDebugLogFlag(enum debug_log_flag flag);
```

?

Reply:
Fixed, thanks.